index.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+require_once __DIR__ . '/Autor.php';
+require_once __DIR__ . '/ImprimirAutor.php';
+require_once __DIR__ . '/Libro.php';
+require_once __DIR__ . '/Revista.php';
+
+
+$autor1 = new Autor('Gardiner ', 'Greene ', 'EEUU', 1827);
```

Comment:
```suggestion
$autorGardinerGreene = new Autor('Gardiner ', 'Greene ', 'EEUU', 1827);
```
El nombre `$autor1` puede tener un nombre más descriptivo.